extensions/ql-vscode/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## 1.5.8 - 2 December 2021
 
+- Add CodeLens feature to make the CodeQL quick evaluation command more accessible. [#1035](https://github.com/github/vscode-codeql/pull/1035)
```

Comment:
This should go in the unreleased section of the changelog.

Reply:
Whoops. I thought I put it there. I'll fix it.